config.mk

```diff
@@ -92,9 +92,10 @@ else ifeq ($(OS),Linux)
 
     ifeq ($(ARCH),x86_64)
       export ENABLEFMA4=1
-      export ENABLEAVX512F= $(shell expr `$(CC) -dumpversion | sed 's/\..*//g'` '>=' 5)
+      export ENABLEAVX512F= $(shell expr `$(CC) -dumpversion | sed 's/\..*//g'` '>=' 6)
       export ENABLEFLOAT80= $(shell expr `$(CC) -dumpversion | sed 's/\..*//g'` '>=' 5)
-      export ENABLEFLOAT128= $(shell expr `$(CC) -dumpversion | sed 's/\..*//g'` '>=' 5)
+      export ENABLEFLOAT128= $(shell expr `$(CC) -dumpversion | sed 's/\..*//g'` '>=' 7)
```

Comment:
yes, this is a mistake, thanks for finding it out. I reverted the change for the FP128 check, the build should still pass.